src/components/shared/forms/styles/pill.scss

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+$pillHeight: 30px;
+$pillMinWidth: 127px;
+$pillIconSize: 14px;
+.pill {
+  display: flex;
+  flex-shrink: 0;
+  justify-content: space-between;
+  border: 0;
+  align-items: center;
+  text-align: left;
+  float: right;
+  white-space: nowrap;
+  margin: 7px 7px 0px 0px;
+  cursor: pointer;
+  font-family: inherit;
+  @include rem-fallback(padding, 0px, $pillHeight/4);
+  @include rem-fallback(min-width, $pillMinWidth);
+  p{
+    padding: 0px;
+    margin: 0px;
+  }
+  div{
+    float:right;
+  }
+
+  .pill-icon {
+    @include rem-fallback(height, $pillHeight);
+    @include rem-fallback(width, $pillIconSize);
+
+    float: right;
+    position: relative;
+    svg { @include square-px-rem($pillIconSize);}
+
+
+    + .pill-text { @include rem-fallback(padding-left, 5px); }
+  }
+
+  .pill-text {
+    @include rem-fallback(padding-right, $pillHeight - $pillIconSize);
+    @include rem-font-size(14px);
+  }
+
+  @include themify($themes) {
+    color: themed('colorBtnText');
+    background-color: themed('colorBtnBackground');
+    border-radius: 3px;
```

Comment:
The border radius should be moved outside the themify since it isn't being themed.

Reply:
Moved.